### ISSUE TITLE
Test_createPackageMap test failing with go1.15

### ISSUE THREAD

**Issue by GrigoriyMikhalkin** [golang/mock#483]
**Actual behavior** 
Test `Test_createPackageMap` fails when run with go1.15. Output:
```
    --- FAIL: Test_createPackageMap/golang_package (0.00s)
        mockgen_test.go:359: createPackageMap() gotPackageName = , wantPackageName = context
        mockgen_test.go:362: createPackageMap() gotOk = false, wantOK = true
    --- FAIL: Test_createPackageMap/third_party (0.00s)
        mockgen_test.go:359: createPackageMap() gotPackageName = , wantPackageName = present
        mockgen_test.go:362: createPackageMap() gotOk = false, wantOK = true
```

**Expected behavior**
When i run tests with go1.14 all tests are passing.

**Comment by kishaningithub** [comment]
Linking here the [PR](https://github.com/golang/mock/pull/479)

**Comment by codyoss** [comment]
This seems to be due to change in go list in 1.15. In older versions of go the command `go list -json context a/bad/path` worked and would return info about context. The same command in 1.15 gives an error:

```
package a/bad/path is not in GOROOT (/usr/local/go/src/a/bad/path)
```

Or in the case of something more real looking module: `go list -json context github.com/a/bad/path`

Output:

```
go: finding module for package github.com/a/bad/path
...
```

In practice this should not be an issue though as the imports been scanned from source code should be resolvable. I will get a fix out for this.

**Comment by codyoss** [comment]
Fixed in #512.
